vispy/visuals/__init__.py

```diff
@@ -22,8 +22,8 @@
 from .isoline import IsolineVisual  # noqa
 from .isosurface import IsosurfaceVisual  # noqa
 from .line import LineVisual  # noqa
-from .line_plot import LinePlotVisual  # noqa
-from .markers import MarkersVisual, marker_types  # noqa
+# from .line_plot import LinePlotVisual  # noqa
+# from .markers import MarkersVisual, marker_types  # noqa
```

Comment:
I need to replay the changes made by #6 (change marker.py) so that this can be uncommented and still work. But, there was the `is` to be changed to `==`. This was presumably fixed in the merge commit cdca89cb537ff0e389b28188e3be533bc5d3d4ea

But, well, that's a _merge commit_. How do I replay a merge commit over regular history?